parsl/executors/high_throughput/executor.py

```diff
@@ -1,34 +1,37 @@
-from concurrent.futures import Future
-import typeguard
+"""HighThroughputExecutor builds on the Swift/T EMEWS architecture to use MPI for fast task distribution
```

Comment:
This must've come in from a bad merge.

Reply:
There was a conflict with master, which I had to resolve by hand, and I might have added what was removed.